portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/VersionsNavigator.tsx

```diff
@@ -400,24 +400,31 @@ const VersionsNavigator = ({
     setSelectedItems(cloneState);
   };
 
-  const renderVersion = (elementIndex: number) => {
-    const item = filteredRecords[elementIndex];
-    const versOrd = versions.length - versions.indexOf(item);
-
+  const rowRenderer = ({
+    key, // Unique key within array of rows
+    index, // Index of row within collection
+    isScrolling, // The List is currently being scrolled
+    isVisible, // This row is visible within the List (eg it is not an overscanned row)
+    style, // Style object to be applied to row (to position it)
+  }: ListRowProps) => {
     return (
       <FileVersionItem
+        style={style}
+        key={key}
         fileName={actualInfo?.name || ""}
-        versionInfo={item}
-        index={versOrd}
+        versionInfo={filteredRecords[index]}
+        index={index}
```

Comment:
Version order is incorrect. Current version shouldn't be 0. 
```suggestion
        index={versOrd}
```

Reply:
done